server/src/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -340,6 +340,7 @@
     private ScheduledExecutorService _executor;
     private ScheduledExecutorService _checkExecutor;
     private ScheduledExecutorService _networkStatsUpdateExecutor;
+    private ExecutorService _routerOobStartExecutor;
```

Comment:
we are trying to get rid of these _'s . no need to adhere to this old convention. In fact you may want to rename the others as well (in a separate commit for review ease) (this is not a request for change, just a suggestion)